data/freesurfer_fsaverage/LICENSE

```diff
@@ -0,0 +1,191 @@
+         FreeSurfer Software License Agreement ("Agreement")
```

Comment:
The title on that license uses the word Software, not data. I think they would fine with us using it as a test data set, but we ought to ask on the FS mailing list to be sure.